service/artifact-life-cycle.rest.xml

```diff
@@ -6,9 +6,16 @@
           name="artifact-life-cycle" displayName="Artifact Life Cycle REST API" version="1.0.0"
           description="Services and entity operations for Artifact Life Cycle">
 
+<!--    <resource name="artifactLogs" require-authentication="anonymous-all">-->
```

Comment:
Why comment? Why not delete? @puru-khedre

Reply:
Removed